Lodgify.Payments.Stripe.Server/Controllers/AccountController.cs

```diff
@@ -1,10 +1,13 @@
-﻿using Lodgify.Payments.Stripe.Application.UseCases.CreateAccount;
+﻿using Lodgify.Authentication.Constants;
+using Lodgify.Payments.Stripe.Application.UseCases.CreateAccount;
 using Lodgify.Payments.Stripe.Server.Requests;
 using MediatR;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
 namespace Lodgify.Payments.Stripe.Server.Controllers;
 
+[Authorize]
```

Comment:
How about making methods from this controller available only for users who are in trial or are subscribed?
Other users should not be able to create Stripe accounts. 
Most likely they should also not be able to access Stripe components (so create the session).
```suggestion
[Authorize(nameof(LodgifyAuthPolicies.AnySubscribed))]
```

Reply:
The same for account session?